<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>configwndClass</class>
 <widget class="QDialog" name="configwndClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>237</width>
    <height>254</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>configwnd</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Program configuration</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="5" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>68</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="checkBox_MinimalColorProcessing">
        <property name="text">
         <string>minimal post-processing</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Camera models</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="comboBox_CameraList"/>
      </item>
      <item row="3" column="0">
       <widget class="QFrame" name="frame_Postprocessing">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="midLineWidth">
         <number>1</number>
        </property>
        <widget class="QCheckBox" name="checkBox_Debayerization">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>101</width>
           <height>18</height>
          </rect>
         </property>
         <property name="text">
          <string>de-bayerization</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="checkBox_SavePreview">
        <property name="text">
         <string>save preview</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>48</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButton_OK">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pushButton_Cancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>comboBox_CameraList</tabstop>
  <tabstop>checkBox_MinimalColorProcessing</tabstop>
  <tabstop>checkBox_Debayerization</tabstop>
  <tabstop>checkBox_SavePreview</tabstop>
  <tabstop>pushButton_OK</tabstop>
  <tabstop>pushButton_Cancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
